### PR TITLE
docathon clarify/simplify example

### PR DESCRIPTION
**Pull request by ConradBraam**
No actual code changes, slightly easier to read and intentions clearer.

### CODE REVIEW

File: mbed-os.lib [hunk 1]
```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/mbed-os/#e1bea44212b8275f7d8ce7253e758c2e25c57482
```
Comment (SeppoTakalo) [inline mbed-os.lib:1]: Why changes to Mbed OS? Remove this.

**Review by SeppoTakalo — CHANGES_REQUESTED** [review]
Changes look good.
Remove the Mbed OS .lib change.

File: mbed-os.lib (hunk 1 above)
Comment (ConradBraam) [reply mbed-os.lib:1]: I never intended this file to be in the commit, not sure how it crept in, I'll be carefull next time, thanks.

**Comment by ConradBraam** [comment]
will redo this.

**Pull request closed without merging**